app/graphql/types/media_reaction_vote.rb

```diff
@@ -0,0 +1,14 @@
+class Types::MediaReactionVote < Types::BaseObject
+  description 'The author who upvoted a specific media reaction.'
+
+  field :id, ID, null: false
+
+  field :author, Types::Profile,
+    null: false,
+    description: 'Profile who upvoted this media reaction.',
+    method: :user
+
+  field :media_reaction, Types::MediaReaction,
+    null: false,
+    description: 'Reaction that was upvoted.'
+end
```

Comment:
Is this type used anywhere or can it be deleted?

Reply:
good catch totally forgot to delete.